monai/visualize/occlusion_sensitivity.py

```diff
@@ -130,168 +145,182 @@ class OcclusionSensitivity:
 
             # Add to list
             batch_images.append(occlu_im)
-            batch_ids.append(class_idx)
 
             # Once the batch is complete (or on last iteration)
             if len(batch_images) == self.n_batch or i == num_required_predictions - 1:
-                # Do the predictions and append to sensitivity map
-                sensitivity_im = _append_to_sensitivity_im(self.nn_module, batch_images, batch_ids, sensitivity_im)
+                # Do the predictions and append to sensitivity maps
+                sensitivity_ims = _append_to_sensitivity_ims(self.nn_module, batch_images, sensitivity_ims)
                 # Clear lists
                 batch_images = []
-                batch_ids = []
-
-        # Subtract baseline from sensitivity so that +ve values mean more important in decision process
-        sensitivity_im = baseline - sensitivity_im
 
         # Reshape to match downsampled image, and unsqueeze to add batch dimension back in
-        sensitivity_im = sensitivity_im.reshape(tuple(downsampled_im_shape)).unsqueeze(0)
+        for i in range(num_classes):
+            sensitivity_ims[i] = sensitivity_ims[i].reshape(tuple(downsampled_im_shape)).unsqueeze(0)
 
-        return sensitivity_im, output_im_shape
+        return sensitivity_ims, output_im_shape
 
     def __call__(  # type: ignore
-        self, x: torch.Tensor, class_idx: Optional[Union[int, torch.Tensor]] = None, b_box: Optional[Sequence] = None
+        self,
+        x: torch.Tensor,
+        b_box: Optional[Sequence] = None,
     ):
         """
-        Args:
-            x: image to test. Should be tensor consisting of 1 batch, can be 2- or 3D.
-            class_idx: classification label to check for changes. This could be the true
-                label, or it could be the predicted label, etc. Use ``None`` to use generate
-                the predicted model.
-            b_box: Bounding box on which to perform the analysis. The output image
-                will also match in size. There should be a minimum and maximum for
-                all dimensions except batch: ``[min1, max1, min2, max2,...]``.
-                * By default, the whole image will be used. Decreasing the size will
-                speed the analysis up, which might be useful for larger images.
-                * Min and max are inclusive, so [0, 63, ...] will have size (64, ...).
-                * Use -ve to use 0 for min values and im.shape[x]-1 for xth dimension.
-        Returns:
-            Depends on the postprocessing, but the default return type is a Numpy array.
-            The returned image will occupy the same space as the input image, unless a
-            bounding box is supplied, in which case it will occupy that space. Unless
-            upsampling is disabled, the output image will have voxels of the same size
-            as the input image.
+        :param x: image to test. Should be tensor consisting of 1 batch; spatially can be 2- or 3D.
```

Comment:
the previous docstring style is consistent with the other files

Reply:
Ok, will sort this tomorrow.